src/main/java/entity/Weapon.java

```diff
@@ -0,0 +1,24 @@
+package entity;
+
+import entity.Equipment;
+
+public class Weapon extends Equipment {
+
+    /**
+     * Creates a new entity.Weapon object.
+     * @param name the name of the entity.Weapon.
+     * @param num the number of weapons.
+     * @param weaponAttackPoint the attack power value.
+     */
+    public Weapon(String name, int num, int weaponAttackPoint){
```

Comment:
Reminder: if we remove the 'num' attribute for Item class, we do not need it as an argument in this constructor